executor/distsql.go

```diff
@@ -68,7 +68,8 @@ type lookupTableTask struct {
 
 	// The handles fetched from index is originally ordered by index, but we need handles to be ordered by itself
 	// to do table request.
-	// The indexOrder map is used to save the original index order for the handles.
+
```

Comment:
The above comment is used to comment `indexOrder`, we should not add an empty line.